README.md

```diff
@@ -245,6 +245,24 @@ postedData.json
 
 * if `postedData.json` doesn't exist on the filesystem when `/match/against/file` is requested, stubby will match post contents against `{"fallback":"data"}` (from `post`) instead.
 
+#### json
+
+* not used if `post` or `file` are present.
+* will be parsed into a JavaScript object.
+* allows you to specify a JSON string that will be deeply compared with a JSON request
+
+Although not required, it is recommended to also specify a `application/json` header requirement.
+
+```yaml
+-  request:
+      url: ^/match/against/jsonString$
+      headers:
+         content-type: application/json
+      json: '{"key1":"value1","key2":"value2"}'
```

Comment:
I actually considered this at length myself.  The reasons I chose this implementation are: 

- The requirement that it be a JSON parsable string ensures that it is an object that can be represented in JSON.  It avoids situations like NaN and Infinity or potentially other members of the YAML superset that can't be represented in JSON.  Granted, not a huge concern perhaps.
- JSON.parsing only has to be done once, during `endpoints.create` for each request that includes a `.json` member.  The incoming requests still have to be parsed, but that is unavoidable.
- It's easier/quicker to copy paste the response from a browser or recording proxy.  Converting a lot of objects from JSON to YAML would take time.

If it's more idiomatic to Stubby4Node to use YAML here, I have no objection, and it's fairly trivial to implement.  Even supporting both would not be hard (check if it's a string or object before parsing).  I'll change it if you like :smile_cat: